indico/util/i18n.py

```diff
@@ -178,6 +178,11 @@ def weekday(self, daynum, short=True):
         """Return the week day given the index."""
         return self.days['format']['abbreviated' if short else 'wide'][daynum]
 
+    @property
+    def html_locale(self):
```

Comment:
There are a few reasons:

1. The way locale is set for moment is a bit hackish. Rather than basing everything on moment, I would base moment on the newly added attribute instead.
2. Creating a hard-dependency on a library feature will prevent you from removing the library in the long run if, for some reason, the library becomes unusable or platform features supersede it. So I wouldn't recommend it. It also decreases the flexibility of the custom elements, which are currently relying only on platform features as a design choice, to ensure longevity of the code.

Let me know if you still insist on using moment.locale() and I'll update the code.

Reply:
> Rather than basing everything on moment, I would base moment on the newly added attribute instead.

I'd be fine with that, but I'm not sure if it'll work straight away. The moment locales are generated based on what moment contains (see generate_moment_locales.py).

Anyway, maybe best to focus on the other things and leave this one for later?

Reply:
`session.moment_lang` cannot be used for this because it's missing the territory suffix afaics.

Reply:
ah indeed, because moment uses just the bare language name for the "main" version of a language (e.g. `fr` for french, and `fr-ca` for canadian french)...